src/components/Card/Card.tsx

```diff
@@ -45,8 +48,55 @@ function Card({
     };
   }, [cardholderRef]);
 
+  const [springs, api] = useSpring(
```

Comment:
api is a very vague name. Type definitions say that `useSpring` returns `[SpringValues, SpringRef]`. so you might call them like that.